src/test/supportMethods/encodeBody.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var htmlSymbols = require('./../../api/dictionary').htmlBodyValues;
+var decode = require('./../../api/decode');
+var moment = require('moment');
+
+/* exports.encodeBody =*/
+// encoding body in the same way as mobile app do this
+// note : encodeBody invokes itself (recursion)
+function encodeBody(body) {
+    var decodedBody = {};
+    for (var iter in body) {
+        // looking for field short name , for HTTP req
+        var decodedKey = htmlSymbols[iter];
```

Comment:
this is for testing current version of data upload from mobile side , it is encodes http body with events in the same way with the same rules as mobile app do that before send data to csl